lib/hammer_cli_foreman/template.rb

```diff
@@ -156,15 +156,37 @@ def print_data(message)
       build_options
     end
 
+    class CloneCommand < HammerCLIForeman::SingleResourceCommand
+      action :clone
 
-    HammerCLIForeman::AssociatingCommands::OperatingSystem.extend_command(self)
-
+      success_message _('Config template cloned')
+      failure_message _('Could not clone the config template')
 
-    autoload_subcommands
+      def self.command_name(name = nil)
+        super(name) || 'clone'
+      end
```

Comment:
Since it's less likely to inherit from this class, it can be simplified to `command_name 'clone'`